tests/myst/exec-not-signed/Makefile

```diff
@@ -33,7 +33,6 @@ test-a:
 	grep -E "argv\[2]=green" result
 	grep -E "argv\[3]=blue" result
 	grep -E "argv\[4]=yellow" result
-	grep -E "TESTNAME=tests/myst/exec-not-signed" result
```

Comment:
We are expecting no environment variables, so create a result file for this test and then do a compare of the results to the expected output... which is probably just the args at this point

Reply:
Added negative testing for this